eos/statistics/log-likelihood.cc

```diff
@@ -1081,43 +1090,36 @@ namespace eos
 
             virtual double sample(gsl_rng * rng) const
             {
-                // Holds the samples in n dimensions
-                std::array<double, n_> random_samples;
+                auto k = _mean->size;
 
-                // generate standard normals
-                std::generate(random_samples.begin(), random_samples.end(), std::bind(gsl_ran_ugaussian, rng));
+                // generate standard normals in observables
+                for (auto i = 0u ; i < k ; ++i)
+                {
+                    gsl_vector_set(_observables, i, gsl_ran_ugaussian(rng));
+                }
 
-                // transform
-                random_samples = chol * random_samples;
+                // transform: observables2 <- _chol * observables
+                gsl_blas_dgemv(CblasNoTrans, 1.0, _chol, _observables, 0.0, _observables_2);
```

Comment:
Here and elsewhere: The Cholesky decomposition is a triangular matrix. Why don't you use `gsl_blas_dtrmv`?

Reply:
In principal a good idea. However, present GSL deprecated the ```gsl_linalg_cholesky_decomp``` function, and I cannot find out easily right now what the correct output is (upper/lower diagonal).
Might be a good idea to open a new issue to keep track of this #70.

Reply:
The output should be lower triangular. But ok, we follow up on this in #70